### ISSUE TITLE
XStore privileges support super type user

### ISSUE THREAD

**Issue by clcc2019** [polardb/polardbx-operator#44]
Request xstore api privileges support super type user,Thanks

**Comment by vettalwu** [comment]
@clcc2019 Do you mean a privilege filed in xstore crd that you can define admin user and password?

**Comment by clcc2019** [comment]
> @clcc2019 Do you mean a privilege filed in xstore crd that you can define admin user and password?

Yes, I defined a management account, but it does not have super permissions, which means I need to manually use root to modify my account permissions.
<img width="562" alt="image" src="https://github.com/polardb/polardbx-operator/assets/46099295/41b145fd-607d-421a-8a00-8eb3bd674ffe">
